### PR TITLE
Jupyter export magic command

### PR DESCRIPTION
**Pull request by tyfiero**
### Describe the changes you have made:

Added a %jupyter magic command to export the current session as a jupyter notebook file, that you can run in Google Collab.

### Pre-Submission Checklist (optional but appreciated):

- [x] I have included relevant documentation updates (stored in /docs)
- [x] I have read `docs/CONTRIBUTING.md`
- [x] I have read `docs/ROADMAP.md`

### OS Tests (optional but appreciated):

- [ ] Tested on Windows
- [x] Tested on MacOS
- [ ] Tested on Linux

### CODE REVIEW

File: interpreter/terminal_interface/magic_commands.py [hunk 1]
```diff
@@ -172,6 +175,79 @@ def handle_count_tokens(self, prompt):
...
     display_markdown_message("\n".join(outputs))
 
+def get_desktop_path():
+    # For Windows
+    if os.name == 'nt':
+        desktop = os.path.join(os.path.join(os.environ['USERPROFILE']), 'Desktop')
+    # For MacOS and Linux
+    else:
+        desktop = os.path.join(os.path.join(os.path.expanduser('~')), 'Desktop')
+    return desktop
```
Comment (Notnaton) [inline interpreter/terminal_interface/magic_commands.py:185]: Should we use the download folder instead of the Desktop?

**Comment by Notnaton** [comment]
Awesome PR @tyfiero 
I have tested it on windows

File: interpreter/terminal_interface/magic_commands.py (hunk 1 above)
Comment (tyfiero) [reply interpreter/terminal_interface/magic_commands.py:185]: Yeah that makes a lot more sense than desktop. I can change that

Comment (tyfiero) [reply interpreter/terminal_interface/magic_commands.py]: Changed it!

**Review by Notnaton — APPROVED** [review]

**Comment by KillianLucas** [comment]
Nice! Great feature idea. It looks like the docs say it saves to the desktop, but I think it's saving to downloads. I also think it would be okay if it saved to cwd, any option works. Thanks @tyfiero!

**Comment by tyfiero** [comment]
Fixed the docs, it should be good to go!

**Comment by KillianLucas** [comment]
Looks good, merging now. Great work @tyfiero!

**Pull request merged**